stairctl/src/main/java/bio/terra/stairctl/commands/ConnectCommands.java

```diff
@@ -31,47 +33,41 @@ public void connect(
               defaultValue = ShellOption.NULL)
           String dbname,
       @ShellOption(
-              value = {"-h", "--host"},
+              value = {"-H", "--host"},
```

Comment:
This was required as `-h` conflicts with `help`, which is a global option. Spring shell didn't report an error but it did cause some odd problems when trying to use other options for this command.